homeassistant/data_entry_flow.py

```diff
@@ -547,11 +548,9 @@ def async_create_entry(
         data: Mapping[str, Any],
         description: str | None = None,
         description_placeholders: Mapping[str, str] | None = None,
-    ) -> FlowResult:
+    ) -> _FlowResultT:
```

Comment:
Should this just return `FlowResult`, then subclasses can specialize? Or is that not allowed?
```suggestion
    ) -> FlowResult:
```

Reply:
Doesn't that require us to overwrite every function when it's an extended version?

Reply:
> Should this just return FlowResult, then subclasses can specialize?

I don't understand this question. The point with the generic type is that subclasses of `FlowHandler` and `FlowManager` specify what subclass of `FlowResult` should be used in the flow. If we don't use the generic here the new type information that the generic gives won't be used. The return value will show as `FlowResult` even though subclasses have specified what subclass of `FlowResult` should be returned.

Reply:
So that means it should stay this way?

Reply:
I think so, but I haven't looked through all the code. There may be more changes required.

Reply:
Haha, I already guessed so. :)